src/pages/search/Search.tsx

```diff
@@ -29,11 +31,65 @@ const Search = () => {
   );
   return (<Fade>
     <Query query={SEARCH_LISTINGS} variables={{ input, ...LISTING_CARD_IMAGE_DIMENSIONS }}>
-      {({ loading, error, data }) =>
-        loading ? <Loading /> :
-        error ? <h2>Error: {error.message}</h2> :
-        <SearchPage listings={data.searchListings} />
-      }
+      {({ loading, error, data }) => {
+        if (loading) {
+          return (
+            <Container tag={Fade} className={VIEWPORT_CENTER_LAYOUT}>
+              <Loading height="8rem" width="8rem" />
+            </Container>
+          );
+        }
+        if (error && error.graphQLErrors[0].extensions && error.graphQLErrors[0].extensions.code === 'BAD_USER_INPUT') {
+          return (
+            <Container tag={Fade} className="d-flex flex-column align-items-center justify-content-center bee-without-header-height-container p-0" fluid>
+              <div
+                className="bg-img-hero d-flex flex-column align-items-center justify-content-center gradient-overlay-half-primary-v1 bee-without-header-height-container w-100"
+                style={{ backgroundImage: `url('https://static.beenest.com/images/app/misc/painted-ladies2.jpg')` }}>
+                <Row className="px-4 mb-4" noGutters>
+                  <h5 className="font-size-md-down-5text-center text-white">Please enter in a city or region to see results.</h5>
+                </Row>
+                <Row className="d-flex justify-content-center px-0 mx-0 bg-white bee-top">
+                  <Col tag={Fade} className="w-100 p-5">
+                    <SearchBar />
+                  </Col>
+                </Row>
+              </div>
+            </Container>
+          );
+        } else if (error) {
+          return (
+            <Container tag={Fade} className={`${VIEWPORT_CENTER_LAYOUT} flex-column bee-without-header-height-container`}>
+              <h2 className="font-size-md-down-5 text-center">{getFriendlyErrorMessage(error)}</h2>
+              <h5 className="text-center">Please contact{' '}
+                <a className="text-primary" href="https://support.beenest.com/" target="_blank">support</a>{' '}
+                if this continues.
+              </h5>
+            </Container>
+          );
+        }
+
+        const { searchListings } = data;
+        if (searchListings.length === 0) {
+          return (
+            <Container tag={Fade} className="d-flex flex-column align-items-center justify-content-center bee-without-header-height-container p-0" fluid>
+            <div
+              className="bg-img-hero d-flex flex-column align-items-center justify-content-center gradient-overlay-half-primary-v1 bee-without-header-height-container w-100"
+              style={{ backgroundImage: `url('https://static.beenest.com/images/app/misc/painted-ladies2.jpg')` }}>
```

Comment:
This looks almost the same as the `Jumbotron` from `Home`; would it make sense to convert these into a shared component? (Or am I missing some of the differences?)

    <Beebotron>
      <h5 className="...">No listing in this area, please try a different city or region.</h5>
    </Beebotron>

Reply:
they have a different height.  i agree that theyre very similar. the main thing i dont like is that we're using bootstrap/theme classes for some places and `style` for others. 

but yes, i think we can benefit from a shared component that accepts class names and styles, and src.  will make in the next PR